.github/workflows/python-tests.yml

```diff
@@ -27,7 +27,14 @@ jobs:
     - name: Lint with flake8
       run: |
         pip install flake8
-        flake8 . --count --max-complexity=10 --statistics
+        flake8 hw_diag/. --count --max-complexity=10 --statistics
+    - name: Move migrations to expected location
+      run: |
+        mkdir /opt/migrations
+        cp -r migrations /opt/migrations/.
+        cp -r alembic.ini /opt/migrations/.
+        sudo mkdir /var/data
+        sudo chmod 777 /var/data
     - name: Unit tests
       run: |
         pip install pytest
```

Comment:
Shall we add pytest-cov to test-requirememts and add

--cov=hw_diag --cov=bigquery --cov-fail-under=70